### PR TITLE
Build publicise github badges page

### PR DESCRIPTION
**Pull request by solazio**
## Done
- _Build publicise github badges page_

## How to QA
- Go to https://charmhub-io-829.demos.haus/[charm-name]/publicise/badges
- Compare to [design](https://app.zeplin.io/project/5fb68befd7ea8e4d263f3c26/screen/602535cfc3299eb18b86ad3b)

## Issue / Card
Fixes #

## Screenshots
image

### CODE REVIEW

**Review by jkfran — APPROVED** [review]
LGTM!

File: templates/publisher/publicise/github_badges.html [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
 
 {% block publicise_content %}
-  <h4>Github badges page</h4>
+  <h4>Promote your charm using an embeddable GitHub badge</h4>
+  <div class="row">
+    <div class="col-2"></div>
+    <div class="col-6">
+      <p>
+        <a href="/{{ package.name }}">
+          <img alt="{{ package.title }}" src="/{{ package.name }}/badge.svg" />
```
Comment (anthonydillon) [inline templates/publisher/publicise/github_badges.html:10]: It's not useful for the alt to just say a title. It should just be empty in this case

File: templates/publisher/publicise/github_badges.html [hunk 2]
```diff
@@ -1,5 +1,37 @@
 {% extends 'publisher/publicise/publicise_layout.html' %}
 
 {% block publicise_content %}
-  <h4>Github badges page</h4>
+  <h4>Promote your charm using an embeddable GitHub badge</h4>
+  <div class="row">
+    <div class="col-2"></div>
```
Comment (anthonydillon) [inline templates/publisher/publicise/github_badges.html:6]: Can you use the [empty columns from Vanilla](https://vanillaframework.io/docs/patterns/grid#empty-columns).

**Review by anthonydillon — COMMENTED** [review]
Few questions in the code

File: templates/publisher/publicise/github_badges.html (hunk 1 above)
Comment (solazio) [reply templates/publisher/publicise/github_badges.html:10]: ok

File: templates/publisher/publicise/github_badges.html (hunk 2 above)
Comment (solazio) [reply templates/publisher/publicise/github_badges.html:6]: yes, makes sense

**Comment by spencerbygraves** [comment]
Thanks @solazio I think Github in the title should have a lowercase "h". Also, the Publicise tab isn't underlined.

**Comment by tbille** [comment]
@spencerbygraves 

> Thanks @solazio I think Github in the title should have a lowercase "h". 

GitHub is with a uppercase H
Charmhub is with a lowercase h 

:upside_down_face:

**Comment by solazio** [comment]
The "publicise" tab underline is fixed. I'm using "GitHub" with capital "H"

**Pull request merged**